app/views/layouts/application.html.erb

```diff
@@ -19,14 +19,17 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="#">Foodies</a>
+          <a class="navbar-brand" href="/homepage/show">Foodies</a>
         </div>
         <!-- Collect the nav links, forms, and other content for toggling -->
         <div class="collapse navbar-collapse" id="custom-nav">
           <ul class="nav navbar-nav navbar-right">
-            <li><a href="/">Home</a></li>
+            <li><a href="/homepage/show">Home</a></li>
             <li><a href="/recipes">Recipes</a></li>
             <li><a href="/masterpieces">Masterpieces</a></li>
+            <% if user_signed_in? %>
+              <li><%= link_to  new_recipe_path do %><span class="glyphicon glyphicon-plus"></span> New recipe <% end %></li>
+            <% end %>
```

Comment:
Shall we put this change in another commit? It doesn't seem to be related to tags